README.md

```diff
@@ -211,6 +211,23 @@ archive { '/nfs/repo/software.zip':
 }
 ```
 
+### Extract Customization
+
+The `extract_flag` or `extract_command` parameters can be used to override the
```

Comment:
Example added for the simple extract_flag usage. The one other use case is if you need a different flag for 7z on windows v.s. another command in linux which can be done as a hash of flags, but in practice I haven't run into it and I'm not sure if it's a feature that we want to promote.